topsort-api-v2.yml

```diff
@@ -466,6 +497,236 @@ components:
           description: >
             Human-readable explanation of or details about the error.
             The string for a given error may change over time; code should not parse or dispatch based on particular values for this field.
+    Placement:
+      type: object
+      required:
+        - path
+      properties:
+        path:
+          type: string
+          description: >
+            URL path of the page triggering the event.
+            This can be obtained in JS using `window.location.pathname`.
+          example: /categories/dairy
+        position:
+          type: integer
+          description: >
+            For components with multiple items
+            (i.e. search results, similar products, etc),
+            this should indicate the index of a given item within that list.
+          minimum: 0
+        page:
+          type: integer
+          description: >
+            For paginated pages, this should indicate which page number triggered the event.
+          minimum: 0
+        pageSize:
+          type: integer
+          description: >
+            For paginated pages this should indicate how many items are in each result page.
+          minimum: 2
```

Comment:
```suggestion
          minimum: 1
```

Reply:
Since these can all be 1, maybe we can set these fields to required? Marketplaces can set them to 1 when they are irrelevant (think banner ads).